pkg/cache/multi_namespace_cache.go

```diff
@@ -29,14 +29,19 @@ import (
 	"k8s.io/client-go/rest"
 	toolscache "k8s.io/client-go/tools/cache"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/internal/objectutil"
 )
 
 // NewCacheFunc - Function for creating a new cache from the options and a rest config
 type NewCacheFunc func(config *rest.Config, opts Options) (Cache, error)
 
+// a new global namespaced cache to handle cluster scoped resources
+var globalCache = "cluster-scope"
```

Comment:
If we reuse the `namespaceToCache` map, I would propose to use an invalid namespace name for the key, so we are sure not to clash with real namespace names.
```suggestion
const globalCache = "_cluster-scope"
```

Reply:
yeah, right. Done